collect_app/src/main/java/org/odk/collect/android/widgets/ExStringWidget.java

```diff
@@ -268,6 +268,11 @@ public void onButtonClick(int buttonId) {
         }
     }
 
+    private void showKeyboard(Context context) {
```

Comment:
You don't need to pass context here you can just use getContext() method one line below.